sass/layout/_headers.scss

```diff
@@ -2,7 +2,7 @@
 
 .channel-header {
     @include flex(0 0 63px);
-    z-index: 15;
+    z-index: 11;
```

Comment:
Looks fine, but I think we can remove this change.

Reply:
Do we need the `.team-sidebar` one? 

@bradjcoughlin has the `#SidebarContainer` taken care of in his PR.

Reply:
@marianunez we do, so that team icons are above the sidebar when dragging.